File: core/src/main/java/com/graqr/threshr/model/redsky/product/VideoFile.java
package com.graqr.threshr.model.redsky.product;

import com.fasterxml.jackson.annotation.JsonProperty;
import io.micronaut.serde.annotation.Serdeable;

@Serdeable
public record VideoFile(
        @JsonProperty("mime_type") String mimeType,
        @JsonProperty("video_height_pixels") String videoHeightPixels,
        @JsonProperty("video_url") String videoUrl) {
}

File: src/main/kotlin/com/graqr/threshr/circleStore/model/product/VideoFile.kt
package com.graqr.threshr.circleStore.model.product

import com.fasterxml.jackson.annotation.JsonProperty
import io.micronaut.serde.annotation.Serdeable

@Serdeable
data class VideoFile(
    @param:JsonProperty("mime_type") val mimeType: String,
    @param:JsonProperty("video_height_pixels") val videoHeightPixels: String,
    @param:JsonProperty("video_url") val videoUrl: String
) {}
